### DISCUSSION TITLE
Login Maske

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#25] [Shipped - Live on production]
User-Feedback:
_Der Eingeladene muss sich ja erstmal ein Konto erstellen. Der Button, auf den viele aber intuitiv klicken ist „Weiter“, nicht „Konto erstellen“._
_Dann bekomme ich den Anruf „das geht nicht…“ Weil sie sich nicht anmelden können, weil sie ja noch gar kein Konto haben…_
 
_Ist ja nur einmalig, dass jeder sich da durchklickt, aber es ist halt auch der erste Eindruck. Und so viel Selbstreflexion haben die wenigsten, um sich einzugestehen, dass sie selbst auf den falschen Button gedrückt haben (das hab ich jetzt nicht laut gesagt 😊)._

Idee:

![Bildschirmfoto 2022-09-26 um 15 47 06](https://user-images.githubusercontent.com/80519295/192297661-a2a10cfd-f39c-4969-a259-47361f6670b8.png)

Direkt bei der Eingabe der Mail Adresse Kontrolle einbauen, ob Konto zu der Mail-Adresse vorhanden ja/nein.
Wenn ja: Button "Weiter" aktiv / Button "Konto erstellen" deaktiviert
Wenn nein: Button "Weiter" deaktiviert / Button "Konto erstellen" aktiv

**Comment by neongrau** [comment]
Muss ich zugeben ich bin da selbst schon mehrfach drauf reingefallen.
